index.html

```diff
@@ -136,6 +136,164 @@ <h2>
         sensing, depth sensing, video analysis, gesture recognition, and so on.
       </p>
     </section>
+    <section>
+      <h2>
+        Model
+      </h2>
```

Comment:
Maybe we can create a subdivide this section in:
- Model 
  - input controls
    - axes
    - buttons
    - touch 
  - output controls

In the Model's section caput we can define the gamepad's properties (basically the first 4 paragraphs) and then we can go into the controls' subsections. WDYT of something like this?
```
//Model
A gamepad is a collection of input controls...
[...]
... and correspond with the same layout.

A [=gamepad=] may have input controls:
- Axis inputs
- button inputs
- touch surfaces

A [=gamepad=] may have output controls like haptic actuators.

// // Input controls
[...]
// // Output controls
```

Reply:
Adding more structure SGTM.